### PR TITLE
Functionality to update Sock permission

### PR DESCRIPTION
**Pull request by badri-pathak**
Signed-off-by: badri-pathak <badri.pathak@ibm.com>

## Pull request checklist

```release-notes

```

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## What is the current behavior?

-

## What is the new behavior?

-

## How risky is this change?
- [ ] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Comment by badri-pathak** [comment]
Tested with driver image : quay.io/badri_pathak/ibm-spectrum-scale-csi-driver:test20

**Comment by Jainbrt** [comment]
@badri-pathak I would suggest to run csi regression once with this change

**Review by deeghuge — DISMISSED** [review]

**Comment by deeghuge** [comment]
@badri-pathak update the regression test run results before merging

File: driver/csiplugin/server.go [hunk 1]
```diff
@@ -97,6 +98,10 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, ids csi.IdentityServer, c
 	if err != nil {
 		klog.Fatalf("Failed to listen: %v", err)
 	}
+	// Change csi.sock file permsion to read and write only
```
Comment (Jainbrt) [inline driver/csiplugin/server.go:101]: please address the typo , permsion -> permission

Comment (badri-pathak) [reply driver/csiplugin/server.go]: Thanks, have updated the comment.

**Comment by badri-pathak** [comment]
Due to signoff issues, created a new PR.

**Pull request closed without merging**